scripts/utils/TagNuGetClientGitRepostoryOnRelease.ps1

```diff
@@ -26,23 +23,19 @@ param
     [Parameter(Mandatory=$True)]
     [string]$VsTargetBranch,
     [Parameter(Mandatory=$True)]
-    [string]$BuildOutputPath,
-    [Parameter(Mandatory=$True)]
-    [string]$BuildInfoJsonFile
+    [string]$ArtifactsDirectory
 )
 
 # Set security protocol to tls1.2 for Invoke-RestMethod powershell cmdlet
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
 # These environment variables are set on the VSTS Release Definition agents.
-$Branch = ${env:BUILD_SOURCEBRANCHNAME}
-$Build = ${env:BUILD_BUILDNUMBER}
 $Commit = ${env:BUILD_SOURCEVERSION}
 
-$NuGetExePath = [System.IO.Path]::Combine($BuildOutputPath, $Branch, $Build, 'artifacts', 'VS15', "NuGet.exe")
+$BuildInfoJsonFile = [System.IO.Path]::Combine($ArtifactsDirectory, 'BuildInfo','buildinfo.json')
+$NuGetExePath = [System.IO.Path]::Combine($ArtifactsDirectory, 'VS15', "NuGet.exe")
```

Comment:
Any plan to change this to VS17 or something more generic?

Reply:
I think we can/should change it something version agnostic, but we'd need to be very thorough. 

https://github.com/NuGet/Home/issues/8070 has details, but the biggest concern is just how many things we'd need to chgange. 

@zivkan Do you want that issue tagged with CI Initiative?